src/components/Pagination/Pagination.tsx

```diff
@@ -1,67 +1,34 @@
 import * as React from "react";
 import bem from "../../utils/bem";
-import Button, { ButtonOptions } from "../Button/Button";
-import Label from "../Label/Label";
-import Select from "../Select/Select";
-import { ButtonIconPositions, ButtonTypes } from "../Button/ButtonTypes";
-import { IconRotationTypes } from "../Icons/IconTypes";
 export interface PaginationProps {
+    /** BlockName for use with BEM. See how to work with blockNames and BEM here: http://getbem.com/introduction/ */
     blockName?: string;
+    /** Additional className for use with BEM. See how to work with blockNames and BEM here: http://getbem.com/introduction/ */
     className?: string;
-    currentValue: string;
+    /** ID that other components can cross reference for accessibility purposes */
+    id?: string;
+    /** Modifiers array for use with BEM. See how to work with modifiers and BEM here: http://getbem.com/introduction/ */
     modifiers?: string[];
-    nextPageHandler: (event: React.MouseEvent) => void;
-    onSelectBlur: (event: React.MouseEvent) => void;
-    onSelectChange: (event: React.MouseEvent) => void;
-    paginationDropdownOptions: string[];
-    previousPageHandler: (event: React.MouseEvent) => void;
+    /** Right hand element that is the next handler, typically a button */
+    nextPage?: React.ReactNode;
+    /** Left hand element that is the previous handler, typically a button */
+    previousPage?: React.ReactNode;
 }
 
-export default function Pagination(props: PaginationProps) {
+export default function Pagination(
+    props: React.PropsWithChildren<PaginationProps>
+) {
     const {
         blockName,
+        children,
         className,
-        currentValue,
+        id,
```

Comment:
is id used?

Reply:
same with blockName, className, modifiers

Reply:
Should also add tests that the component does apply id, className, etc through if they're passed in.

Reply:
I've removed id, but blockName, className, and modifiers allow consumers to change aspects of Pagination for their app and shouldn't be removed.

I think that making sure blockName, className, and modifiers are applied to components might belong better with the BEM helper, OR we should add that test to every component in the library.

Reply:
if a blockName gets passed in here, where is it applied?  Sorry I feel like I'm missing something very obvious

Reply:
L33, no?

Reply:
GAAAAH github hid literally just these two lines T_T.  Yes is perfect.